From 9974ea5502f31e604af73acdfb8052d9177541cd Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Wed, 25 Oct 2023 16:53:20 +0200
Subject: [PATCH 383/841] fix(st-clock): make _clk_get_ops static

Issue found with sparse tool:
drivers/st/clk/clk-stm32-core.c:227:28: warning: symbol '_clk_get_ops'
 was not declared. Should it be static?

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 41cf6136701e2329367a9a333af8d1bc8c2be542 fix(st-clock): make _clk_get_ops static]
Change-Id: Ia89b33f28762c2e925f0ff08c1ef5fcf3112d2fa
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/338360
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 drivers/st/clk/clk-stm32-core.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/st/clk/clk-stm32-core.c b/drivers/st/clk/clk-stm32-core.c
index 8a7a74242..1d4e91193 100644
--- a/drivers/st/clk/clk-stm32-core.c
+++ b/drivers/st/clk/clk-stm32-core.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2022, STMicroelectronics - All Rights Reserved
+ * Copyright (C) 2022-2023, STMicroelectronics - All Rights Reserved
  *
  * SPDX-License-Identifier: GPL-2.0+ OR BSD-3-Clause
  */
@@ -224,7 +224,7 @@ const struct clk_stm32 *_clk_get(struct stm32_clk_priv *priv, int id)
 	return NULL;
 }
 
-const struct stm32_clk_ops *_clk_get_ops(struct stm32_clk_priv *priv, int id)
+static const struct stm32_clk_ops *_clk_get_ops(struct stm32_clk_priv *priv, int id)
 {
 	const struct clk_stm32 *clk = _clk_get(priv, id);
 
-- 
2.47.2

